cypress/fixtures/b2b-mp/yves/product/dynamic-publish-and-synchronize-dms.json

```diff
@@ -0,0 +1,112 @@
+{
+  "data": {
+    "type": "dynamic-fixtures",
+    "attributes": {
+      "synchronize": true,
+      "operations": [
+        {
+          "type": "transfer",
+          "name": "LocaleTransfer",
+          "key": "locale",
+          "arguments": { "id_locale": 66, "locale_name": "en_US" }
+        },
+        {
+          "type": "helper",
+          "name": "haveStore",
+          "key": "store",
+          "arguments": [{ "name": "TEST_STORE" }]
```

Comment:
That's looks wrong:
- login as user
- create product
- login as a merchant user
- create merchant product
- trigger P&S

b2c, b2b, suite - concrete products
b2c-mp, b2b-mp - merchant products

Reply:
Discussed: Tech debt should be created.

Reply:
https://spryker.atlassian.net/browse/FRW-9465